### ISSUE TITLE
How to read mzTab and mzTab-M stuff

### ISSUE THREAD

**Issue by sneumann** [rformassspectrometry/RforMassSpectrometry#28]
Hi, MSnbase can read mzTab and since this year mzTab-M. 
I'd love to work on some file formats in the RforMassSpectrometry world 
also together with @michaelwitting. How would that work ? 

MTD section goes into `MsExperiment`, which keeps the `SMF` section in `QFeatures`.
If there are associated mzML, MSP or MGF with MS/MS spectra for these features, 
they would go into `Spectra`.

So I would do a Backend for `MsExperiment` which does the above for e.g. `mzTab-M`
and a different one for MetaboLights (ISA-Tab + MAF files). Correct ? 
Do you have naming conventions for this backend ? MsBackendMzTab-M ? MsBackendMetaboLights ?

Yours, 
Steffen

**Comment by lgatto** [comment]
No, there is currently no intention (and, as fas as I can see no need) for backends in `MsExperiment`. `mzTab*` files would get parsed into `QFeatures` objects.

**Comment by lgatto** [comment]
See [MsIO](https://github.com/rformassspectrometry/MsIO)
